src/bot/integrations/discord.ts

```diff
@@ -371,6 +386,51 @@ class Discord extends Integration {
     } catch (e) {
       warning(e.stack);
     }
+
+    try{
```

Comment:
please use proper coding style
```suggestion
    try {
```